app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -29,9 +33,34 @@ const Entry = ({
 }): JSX.Element => {
   const [currentWeek, setCurrentWeek] = useState<IGameWeek['week']>(1);
   const [entries, setEntries] = useState<IEntry[]>([]);
+  const [leagueName, setLeagueName] = useState<string>('');
   const [loadingData, setLoadingData] = useState<boolean>(true);
+  const [survivors, setSurvivors] = useState<number>(0);
+  const [totalPlayers, setTotalPlayers] = useState<number>(0);
   const { user } = useDataStore((state) => state);
 
+  /**
+   * Fetches the current league name.
+   * @returns {Promise<void>}
+   */
+  useEffect(() => {
+    /**
+     * Fetches the current league name.
+     */
+    const getCurrentLeagueName = async (): Promise<void> => {
+      try {
+        const league = await getCurrentLeague(leagueId);
+        setLeagueName(league.leagueName);
+        setSurvivors(league.survivors.length);
+        setTotalPlayers(league.participants.length);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    getCurrentLeagueName();
+  }, [leagueId]);
```

Comment:
Out of curiousity, if you remove `leagueId` as a dependency here, what happens?

Reply:
> Out of curiousity, if you remove `leagueId` as a dependency here, what happens?

I don't know, pull my branch and find out 😜

I'll experiment and let you know.

Reply:
> Out of curiousity, if you remove `leagueId` as a dependency here, what happens?

@choir27 Interestingly, it seems to work just fine 🤔. Looking back at my code, it makes sense that it works fine, thank you for the good question! The latest commit removes it as a dependency.